CRITICAL - 2024-02-03 16:06:20 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
ERROR - 2024-02-03 15:09:18 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db-perpustakaan', 3306, '', 0)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelPengaturan.php(27): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Home.php(22): App\Models\ModelPengaturan->Slider()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelPengaturan.php(27): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Home.php(22): App\Models\ModelPengaturan->Slider()
#5 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#10 {main}
CRITICAL - 2024-02-03 15:09:18 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'db-perpustakaan'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_slider`
WHERE `id_slider` = :id_slider:', [...], false)
 3 APPPATH\Models\ModelPengaturan.php(27): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(22): App\Models\ModelPengaturan->Slider()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
INFO - 2024-02-03 15:09:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-03 15:10:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-03 15:14:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-03 15:14:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-02-03 15:14:59 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db-perpustakaan', 3306, '', 0)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Auth->Register()
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
#5 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Auth->Register()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#10 {main}
CRITICAL - 2024-02-03 15:14:59 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'db-perpustakaan'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_kelas`
ORDER BY `id_kelas` DESC', [], false)
 3 APPPATH\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->Register()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2024-02-03 15:17:21 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db-perpustakaan', 3306, '', 0)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Auth->Register()
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
#5 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Auth->Register()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#10 {main}
CRITICAL - 2024-02-03 15:17:21 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'db-perpustakaan'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_kelas`
ORDER BY `id_kelas` DESC', [], false)
 3 APPPATH\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->Register()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2024-02-03 15:19:55 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db-perpustakaan', 3306, '', 0)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Auth->Register()
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'db-perpustakaan' in C:\xampp 8.1\htdocs\perpuss\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp 8.1\htdocs\perpuss\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp 8.1\htdocs\perpuss\app\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp 8.1\htdocs\perpuss\app\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
#5 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(934): App\Controllers\Auth->Register()
#6 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp 8.1\htdocs\perpuss\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp 8.1\htdocs\perpuss\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp 8.1\htdocs\perpuss\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp 8.1\\ht...')
#10 {main}
CRITICAL - 2024-02-03 15:19:55 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'db-perpustakaan'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_kelas`
ORDER BY `id_kelas` DESC', [], false)
 3 APPPATH\Models\ModelKelas.php(13): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Auth.php(110): App\Models\ModelKelas->AllData()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->Register()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
